org.mango3d.LycheeSlicer.yml

```diff
@@ -0,0 +1,56 @@
+app-id: org.mango3d.LycheeSlicer
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: lycheeslicer
+tags:
+  - proprietary
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --device=dri
+  - --filesystem=home
+modules:
+  - name: lycheeslicer
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/opt
+      - mkdir -p /app/bin
+      - mkdir -p /app/lib
+      - ARCH_TRIPLE=$(gcc --print-multiarch) && cp /usr/lib/${ARCH_TRIPLE}/libbfd-*.so /app/lib
+      - ARCH_TRIPLE=$(gcc --print-multiarch) && cp /usr/lib/${ARCH_TRIPLE}/libsframe.so.1 /app/lib
+      - install -D /usr/bin/ar /app/bin/ar
+      - install -Dm644 ${FLATPAK_ID}.metainfo.xml /app/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - install -D ${FLATPAK_ID}.png /app/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      - install -D ${FLATPAK_ID}.desktop /app/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm755 lycheeslicer /app/bin/lycheeslicer
+      - install -Dm755 apply_extra /app/bin/apply_extra
+    sources:
+      - type: extra-data
+        url: https://mango-lychee.nyc3.cdn.digitaloceanspaces.com/LycheeSlicer-5.4.0.deb
+        filename: lychee.deb
+        sha256: 42f734e4b12846b87f86440c7eb76af13939e805436b3128228184136609b93b
+        size: 117248778
+        only-arches:
+          - x86_64
+      - type: file
+        path: libbfd-2.41.0.so
```

Comment:
not sure where this come from but it surely isn't here

Reply:
just removed it, i forgot i left that in